gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -252,6 +258,20 @@ def run(self) -> GtsfmData:
                 gt_scene_mesh=self.loader.get_gt_scene_trimesh(),
             )
 
+            if self.scene_optimizer.correspondence_augmenter:
```

Comment:
@ayushbaid would it be cleaner to get the two set of correspondences, and merge their keypoints with the KeypointAggregator, and then run the two-view estimator once? Don't we only want to estimate (R,t) once with all the correspondences?

Reply:
I thought about that approach too, but I am wondering if having _lower_ quality correspondences before 2-view estimator will result in poor pose estimates. What do you think?